tran.go

```diff
@@ -16,6 +16,15 @@ const (
 	tmSaveXact      = 9
 )
 
+const (
+	isolationUseCurrent = 0
```

Comment:
perhaps assign it a type of uint8 or `type isoLevel uint8` and use throughout, one less cast in one place.